lib/metadata.js

```diff
@@ -222,6 +224,7 @@ function get(req, res, next) {
             }
 
             if (Object.keys(md).length) {
+                delete md.affinity;
                 delete md.locality;
                 Object.keys(md).forEach(function (k) {
                     if (/_pw$/.test(k)) {
```

Comment:
Do we need further mods to inside the `if (/_pw$/.test(k)) {` branch?  Or did 770 take care of that?